lib/pemverifier.ts

```diff
@@ -14,19 +14,12 @@
  * limitations under the License.
  */
 
-'use strict';
+import * as crypto from 'crypto';
 
-var crypto = require('crypto');
-
-function PemVerifier() {
-  this.verify = function(pubkey, data, signature, encoding) {
-    var verifier = crypto.createVerify('sha256');
+export default function PemVerifier() {
+  this.verify = (pubkey, data, signature, encoding) => {
+    const verifier = crypto.createVerify('sha256');
```

Comment:
Do we want to use named imports for situations like this where only a few operations are needed?